rust/entity/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
we just need the Entity crate for the Models right? I think we should be able to delete the main.rs here

Reply:
Yeah, I think this was generated by `cargo init`, I'll delete it!